tests/test_skvbc_persistence.py

```diff
@@ -558,12 +557,12 @@ def test_st_when_primary_crashes_once(self):
             )
 
             stable_replicas = \
-                tuple(set(range(n)) - {current_primary, stale})
+                bft_network.all_replicas(without={current_primary, stale})
 
         bft_network.start_replica(stale)
 
         await bft_network.wait_for_state_transfer_to_stop(
-            up_to_date_node=random.choice(stable_replicas),
+            up_to_date_node=current_primary,
```

Comment:
This change may significantly improve the intermittent failures reported here:
https://github.com/vmware/concord-bft/issues/294